File: Back/src/Fit.Domain/Models/TreinoExercicio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Fit.Domain.Models
{
    public class TreinoExercicio
    {
        [Key]
        [Column(Order = 0)]
        public int TreinoId { get; set; }
        public Treino? Treino { get; set; }
        [Key]
        [Column(Order = 1)]
        public int ExercicioId { get; set; }
        public Exercicio? Exercicio { get; set; }  
        public int UnidadeId { get; set; }
        public Unidade? Unidade { get; set; }
        public int DiaDaSemana { get; set; }
        public int Valor { get; set; }
        public int Series { get; set; }
        public int Repeticoes { get; set; }

    }
}

File: Front/Fit-App/src/app/models/TreinoExercicio.ts
import { Exercicio } from "./Exercicio";
import { Treino } from "./Treino";
import { Unidade } from "./Unidade";

export interface TreinoExercicio {
  TreinoId: number;
  Treino: Treino;
  ExercicioId: number;
  Exercicio: Exercicio;
  UnidadeId: number;
  Unidade: Unidade;
  DiaDaSemana: number;
  Valor: number;
  Series: number;
  Repeticoes: number;
}
